sphinx/builders/__init__.py

```diff
@@ -84,6 +84,10 @@ class Builder:
     #: The builder supports data URIs or not.
     supported_data_uri_images = False
 
+    # basename of output assets directories
+    imagedir = ""
+    sourcesdir = ""
```

Comment:
Is `sourcedir` a common attribute of builders? I don't think so.

Reply:
Only used in html though...

Reply:
If so, no reason to add it to the common Builder class.